irohad/consensus/yac/impl/yac.cpp

```diff
@@ -138,85 +138,101 @@ namespace iroha {
 
       void Yac::closeRound() { timer_->deny(); }
 
+      nonstd::optional<model::Peer> Yac::findPeer(const VoteMessage &vote) {
+        auto peers = cluster_order_.getPeers();
+        auto it =
+            std::find_if(peers.begin(), peers.end(), [&](const auto &peer) {
+              return peer.pubkey == vote.signature.pubkey;
+            });
+        return it != peers.end() ? nonstd::make_optional(*it) : nonstd::nullopt;
+      }
+
       // ------|Apply data|------
 
-      void Yac::applyCommit(model::Peer from, CommitMessage commit) {
+      void Yac::applyCommit(nonstd::optional<model::Peer> from,
```

Comment:
Is from unused?

Reply:
Yes, since now only individual votes are signed.